products/sle/main.pm

```diff
@@ -287,7 +288,13 @@ if (get_var('DEV_IMAGE')) {
 
 # This is workaround setting which will be removed once SCC add repos and allows adding modules
 # TODO: remove when not used anymore
-if (get_var('ALL_MODULES') && sle_version_at_least('15')) {
+if (my $allmodules = get_var('ALL_MODULES') && sle_version_at_least('15')) {
```

Comment:
I prefer to be more explicit about the type of variables. Better add another specific variable when you want to set the variables and keep the existing one as boolean `ALL_MODULES`. I don't think ALL_MODULES=module1,module2 is easy to understand

Reply:
Idea is that this is temporary, so I didn't want to check multiple variables. But yeah, let's make it cleaner, as we know that temporary changes remain forever =)

Reply:
Replaced with WORKAROUND_MODULES variable